kingfisher_scrapy/base_spider.py

```diff
@@ -367,14 +366,15 @@ def start_requests(self):
     encoding = 'utf-8'
     skip_pluck = 'Archive files are not supported'
     compressed_file_format = None
-    archive_format = 'zip'
     file_name_must_contain = ''
 
     @handle_http_error
     def parse(self, response):
+        archive_name, archive_format = os.path.splitext(response.request.meta['file_name'])
+        archive_format = archive_format.replace('.', '')
```

Comment:
```suggestion
        archive_format = archive_format[1:].lower()
```

Works even if `archive_format = ''`. Handles a potential future case where the filename from the source is capitalized.